ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlQueryResponseImpl.java

```diff
@@ -198,4 +201,11 @@ public String getId() {
   public Status getStatus() {
     return status;
   }
+
+  public void addToWarnings(List<String> unaddedWarnings) {
+    if (unaddedWarnings != null && !unaddedWarnings.isEmpty()) {
+      warnings.addAll(
+          unaddedWarnings.stream().filter(Objects::nonNull).collect(Collectors.toSet()));
+    }
+  }
```

Comment:
Should I write tests for this method? It doesn't do much, but tests for when `addedWarnings` takes a null or empty value would provide value. I would have added those tests already, except that I couldn't find `CqlQueryResponseImplTest.java`, and I don't know that this method justifies the creation of `CqlQueryResponseImplTest.java` for maybe three tests which wouldn't provide much value.

Reply:
Regarding your comment about the overhead of a new class, for production code I would try to avoid it, but for test code I think it's a moot point.